### PR TITLE
GCC 6 fixes

### PR DESCRIPTION
**Pull request by alarcher**

### CODE REVIEW

File: components/mail/evolution/Makefile [hunk 1]
```diff
@@ -25,11 +25,11 @@ COMPONENT_ARCHIVE_HASH=	\
...
 
-include $(WS_TOP)/make-rules/prep.mk
-include $(WS_TOP)/make-rules/configure.mk
-include $(WS_TOP)/make-rules/ips.mk
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/configure.mk
+include $(WS_MAKE_RULES)/ips.mk
 
-PATH=/usr/gnu/bin:/usr/bin:/usr/perl5/bin
+PATH=$(PATH.gnu):/usr/perl5/bin
```
Comment (AndWac) [inline components/mail/evolution/Makefile]: I think you can omit :/usr/perl5/bin because PATH:gnu already includes $(PERL5BINDIR).

Comment (alarcher) [reply components/mail/evolution/Makefile]: Fixed.

**Review by AndWac — APPROVED** [review]
I cannot comment on the patches but the rest looks reasonable to me.

**Comment by alarcher** [comment]
@AndWac Thanks for reviewing :)

**Comment by alarcher** [comment]
@pyhalov @xen0l If you have any comment.

**Review by xen0l — APPROVED** [review]
LGTM

**Pull request merged**